### DISCUSSION TITLE
How to update druid DataSource.

### DISCUSSION THREAD

**Discussion by rupeshpyeole** [Sunbird-Ed/Community#594] [General Discussion]
Hi everyone, we want to know which job runs to update "ml-survey-status" druid DataSource?
Here we are facing issue is, we are not getting data in "ml-survey-status" bucket and hence data is not coming in druid DataSource.
I, kindly request you to help us on this finding job for this druid data source or root cause of this issue.

Thanks and regards,
Rupesh

**Comment by gandham-santhosh** [comment]
@Shakthieshwari  @aks30 please check this

**Comment by adithyadinesh0412** [comment]
Hi @rupeshpyeole ,

Kindly run `ml-analytics-service/survey/pyspark_survey_rollup.py` for updating  `ml-survey-status` datasource. 
Please let us know if you have any further queries or concerns.

**Reply by adithyadinesh0412** [reply]
Hi @rupeshpyeole ,
Sorry for the confusions. Please ignore my previous comment.
This is a known issue and currently we don't have any scripts to update  `ml-survey-status` datasource.
We are working to bring a fix for this in the `6.0.0` release.
PFB the Jira Ticket :
[ED-1383](https://project-sunbird.atlassian.net/browse/ED-1383)
